src/new-client/src/app/lib/components/site-list.test.tsx

```diff
@@ -13,6 +13,23 @@ describe('<SiteList>', () => {
     expect(screen.queryByRole('link', { name: 'Site Beta' })).toBeVisible();
   });
 
+  it('renders sites in ascending alphabetical order', async () => {
+    const testSites: Site[] = [
```

Comment:
There's already a `mockSites` object we can import from `@testing/data` we could use instead?

Reply:
As discussed, I will just leave this as I have purposefully set up sites out of alpha order to check sorting which is different to the mocksite being used which will force changes to other tests.